packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -424,8 +424,9 @@ async function load_shadow_data(route, event, prerender) {
 			data.body = body;
 		}
 
-		if (mod.get) {
-			const result = await mod.get.call(null, event);
+		const resultHandler = (method === 'head' && mod.head) || mod.get;
+		if (resultHandler) {
+			const result = await resultHandler.call(null, event);
```

Comment:
`resultHandler` feels a bit clunky (it suggests the other `handler` is some _other_ kind of handler). I suggest we just call it `get`. Also, the `.call(null, ` is only necessary with dynamic dispatch

```suggestion
		const get = (method === 'head' && mod.head) || mod.get;
		if (get) {
			const result = await get(event);
```